php/class-widget-posts.php

```diff
@@ -723,10 +723,7 @@ function get_widget_instance_numbers( $id_base ) {
 		/** @var \wpdb $wpdb */
 		global $wpdb;
 
-		$cache_get = function_exists( 'wpcom_vip_cache_get' ) ? 'wpcom_vip_cache_get' : 'wp_cache_get';
-		$cache_set = function_exists( 'wpcom_vip_cache_set' ) ? 'wpcom_vip_cache_set' : 'wp_cache_set';
```

Comment:
Yes they do. The wpcom_vip versions are actually discouraged now by VIP
unless they identify particular use cases.